src/yaca_lib.rs

```diff
@@ -0,0 +1,162 @@
+use libc::{c_char, c_int, c_void, size_t};
+
+
+#[allow(dead_code)]
```

Comment:
I think this can be already removed, was just a intermediate warning killer. All those functions should be used now.

Reply:
Indeed they are.